pkg/distributor/query_test.go

```diff
@@ -25,10 +25,146 @@ import (
 
 	ingester_client "github.com/grafana/mimir/pkg/ingester/client"
 	"github.com/grafana/mimir/pkg/mimirpb"
+	"github.com/grafana/mimir/pkg/querier/api"
 	"github.com/grafana/mimir/pkg/querier/stats"
 	"github.com/grafana/mimir/pkg/util/limiter"
 )
 
+func TestDistributor_QueryExemplars(t *testing.T) {
+	const numIngesters = 5
+
+	now := model.Now()
+
+	fixtures := []mimirpb.PreallocTimeseries{
+		// Note: it's importa to write at least a sample, otherwise the exemplar timestamp validation doesn't pass.
```

Comment:
```suggestion
		// Note: it's important to write at least a sample, otherwise the exemplar timestamp validation doesn't pass.
```

Reply:
Ooops. Fixed in https://github.com/grafana/mimir/pull/7393/commits/21a259b7c786c23bf835a87f58b0fc1d3a4c1288